### PR TITLE
[PUB-3768] adds tracker to date navigation

### PR DESCRIPTION
**Pull request by jezebelbaal**
### Description

This PR adds a tracker to calendar navigation arrows

### Context & Notes
[(https://buffer.atlassian.net/browse/PUB-3768](https://buffer.atlassian.net/browse/PUB-3768)

### Best Practices Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [ ] I have tested different user stories.
-   [x] I have identified similar or related features and tested they are still working.
-   [x] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

### ⚠️ Pre-deploy Checklist

- [ ] I have manually loaded the URL of the staging deployment and confirmed it's working (see below**)
- [ ] I have confirmed that the Cypress E2E test is passing (_Note: we have a known issue where it doesn't run on some PRs. Hopefully this will be fixed soon._)
- [ ] I have confirmed that all other tests are passing.

** To visit the URL of the staging deployment, please vlick the link in the bot comment, or look under checks at the bottom of this PR and click "Details" next to `bufferbotbrains/cicd-buffer-publish`
image

### CODE REVIEW

**Review by anafilipadealmeida — COMMENTED** [review]
Hey @jezebelbaal , this is looking good! You are just missing the tracking for the OnTodayClick!

**Review by anafilipadealmeida — CHANGES_REQUESTED** [review]

File: packages/calendar/components/ChannelFilter/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
```
Comment (anafilipadealmeida) [inline packages/calendar/components/ChannelFilter/index.js:1]: Is this file from this PR?

File: packages/calendar/index.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { connect } from 'react-redux';
 import { replace } from 'connected-react-router';
 import { calendarWeek, calendarMonth } from '@bufferapp/publish-routes';
+import { actions as analyticsActions } from '@bufferapp/publish-analytics-middleware';
```
[the remark this answers is not in the document]
Comment (jezebelbaal) [reply packages/calendar/index.js:4]: you're right. thanks, Ana!

File: packages/calendar/components/ChannelFilter/index.js (hunk 1 above)
Comment (jezebelbaal) [reply packages/calendar/components/ChannelFilter/index.js]: Nope! Sorry for that. 😅

**Review by anafilipadealmeida — APPROVED** [review]
🚀 🚀

**Review by hamstu — APPROVED** [review]
Looks great – nice work @jezebelbaal 😄

**Pull request merged**